package.json

```diff
@@ -3,23 +3,22 @@
   "version": "0.1.0",
   "private": true,
   "devDependencies": {
```

Comment:
Fixed that web could not be displayed due to ambiguity in version specification.